client/my-sites/checkout/upsell-nudge/index.jsx

```diff
@@ -181,6 +181,7 @@ export class UpsellNudge extends React.Component {
 		}
 	}
 
+
```

Comment:
We need to remove this line, since it's giving a lint error.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/43199/commits/1b1f72a08cbbf1c177aeb1f49716948b234ed1ee